testing/src/main/java/org/oppia/android/testing/AssertionHelper.kt

```diff
@@ -0,0 +1,24 @@
+package org.oppia.android.testing
+
+import org.junit.Assert.fail
+import kotlin.reflect.KClass
+import kotlin.reflect.full.cast
+
+object AssertionHelper {
```

Comment:
You will need to add a new ``kt_android_library`` for this class in the ``testing/BUILD.bazel`` file. I think will look like this:

```starlark
kt_android_library(
    name = "assertion_helpers",
    srcs = ["src/main/java/org/oppia/android/testing/AssertionHelpers.kt"],
    visibility = ["//visibility:public"],
    testonly = True,
    deps = [
        artifact("org.jetbrains.kotlin:kotlin-reflect"),
        artifact("junit:junit:4.12"),
    ]
)
```

Note that you will need to update the TEST_DEPS for the testing directory (here: https://github.com/oppia/oppia-android/blob/develop/testing/BUILD.bazel#L38) to reference the new helper (e.g. by adding ``":assertion_helpers"`` to the list), otherwise the updated test above will fail when running it.

Reply:
Done.